app/analysis/analysis.html

```diff
@@ -1,9 +1,9 @@
 <ul class="nav nav-pills">
   <li ng-class="{ active: $routeSegment.contains('report') }">
-    <a href="#{{ $routeSegment.getSegmentUrl('objects.report') }}">Report</a>
+    <a href="#{{ $routeSegment.getSegmentUrl('objects.report') }}">{{ "Report" | translate }}</a>
```

Comment:
Same question as in transfer-browser - Why `<a href="#{{ $routeSegment.getSegmentUrl('objects.report') }}">{{ "Report" | translate }}</a>`  instead of `<a href="#{{ $routeSegment.getSegmentUrl('objects.report') }}" translate>Report</a>` I think they're equivalent.